client/actions/planning.js

```diff
@@ -249,6 +249,14 @@ const closePlanningEditor = () => (
     { type: 'CLOSE_PLANNING_EDITOR' }
 )
 
+const openAdvancedSearch = () => (
```

Comment:
This is the event's search here, isn't it ? I would move this to the event actions module then.